pymc3_models/models/NaiveBayes.py

```diff
@@ -0,0 +1,283 @@
+import functools as ft
+
+import numpy as np
+import pymc3 as pm
+import scipy.stats
+from sklearn.metrics import accuracy_score
+import theano
+
+from pymc3_models.exc import PyMC3ModelsError
+from pymc3_models.models import BayesianModel
+
+
+__all__ = ["GaussianNaiveBayes"]
```

Comment:
Since we're importing the models in the __init__ file, I don't think we need to keep this.

Reply:
It is not necessary, but I think it improves readability when there are several classes in the same file. I can remove it for now, and put it back If we ever add another class (MultinomialNaiveBayes, etc.) ?

Reply:
sound good!

Reply:
Done.